README.md

```diff
@@ -1,12 +1,24 @@
-# Authors
-- Anssi Kostiainen (Intel)
-- Belem Zhang (Intel)
-- Wei Wang (Intel)
-
 # Local Peer-to-Peer API
 
-For the latest proposals, see the [Local Peer-to-Peer API Explainer](EXPLAINER.md).
+[Local Peer-to-Peer](https://WICG.github.io/local-peer-to-peer/) is a Web platform API proposal for local communication between browsers without the aid of a server.
+
+```js
+let session = await navigator.lp2p.findPeers();
+
+await peer.send("Hello there!");
+```
```

Comment:
```js
let peers = await navigator.lp2p.findPeers();
let peer = peers[0];
await peer.connect();
await peer.send("Hello there!");
```
Is this better? Because we can find more than one peers, we can choose one to connet and send message,